source/pages/Home/Panel/ActivityPanel.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
these file changes are in another PR I don't know if we need to replicate them here.